src/header/messages.ts

```diff
@@ -0,0 +1,36 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
```

Comment:
[nit] It's better to have a common way with adding of avoiding `.` symbol in the end of the description text.